core/src/ai/ai.ts

```diff
@@ -80,3 +69,50 @@ export const getBlockedTiles = (player: Player, board: Board) => {
   });
   return tiles;
 };
+
+export const findBestPath = (player: Player, board: Board) => {
+  const graph = createGraph(board, player);
+
+  for (const lines of board) {
+    for (const tile of lines) {
+      if (!tile) continue;
+      const { x, y } = findTile(board, tile);
+      const siblings = findSiblings(board, {
+        x,
+        y,
+      });
+      for (const key in siblings) {
+        const sibling = siblings[key];
+        if (!sibling) continue;
+
+        let weight = 1;
+        if (sibling?.playedBy === switchPlayer(player)) weight = 2;
```

Comment:
If the tile is already played by the other player, you should not add an edge here otherwise the algorithm will try that tile in the path

Reply:
Yes, I'am aware with it, this was just for testing